Tasks/PackerBuild/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "commonjs",
+        "sourceMap": true,
+        "sourceRoot": "E:\\github\\vsts-tasks\\Tasks\\PackerBuild" 
```

Comment:
No this is not necessary. however, it provides easy reference to start debugging.